src/modm/board/disco_f469ni/module.lb

```diff
@@ -39,10 +39,16 @@ def build(env):
     env.outbasepath = "modm/src/modm/board"
     env.substitutions = {
         "with_logger": True,
-        "with_assert": env.has_module(":architecture:assert")
+        "with_assert": env.has_module(":architecture:assert"),
+        "touch_address": 0x38 if env.get(":disco-f469ni", "b-03") == "b-03" else 0x2a,
```

Comment:
Are you sure revision _B-02_ also has touch controller address `0x38`?

Reply:
I couldn't find any mention of hardware revisions in the official kit PDF, but I found the [two addresses in the CubeHAL BSP](https://github.com/STMicroelectronics/STM32CubeF4/blob/2f3b26f16559f7af495727a98253067a31182cfc/Drivers/BSP/STM32469I-Discovery/stm32469i_discovery.h#L231-L239), but they [detect the driver address at runtime without mentioning revision](https://github.com/STMicroelectronics/STM32CubeF4/blob/2f3b26f16559f7af495727a98253067a31182cfc/Drivers/BSP/STM32469I-Discovery/stm32469i_discovery_ts.c#L173-L182).

So I don't know and I cannot check it…

Reply:
Then I would prefer to define a B-02 revision which we do not know if it exists at all and which we have not tested.

Reply:
Then I would prefer to not define a B-02 revision which we do not know if it exists at all and which we have not tested.